docs/hub-deployment-guide/configure-auth/cilogon.md

```diff
@@ -113,7 +113,7 @@ The steps to enable the JupyterHub CILogonOAuthenticator for a hub are simmilar
               CILogonOAuthenticator:
                 # This config option will only display GitHub as the only identity provider option
                 shown_idps:
-                  - https://github.com/login/oauth/authorize
+                  - http://google.com/accounts/o8/id
```

Comment:
I don't think this should be changed given that it states the following:

*"This example uses GitHub as the only identity provider to show to the user."*

Is there any reason for which you'd wish to exemplify with Google instead of GitHub?

Reply:
Thanks for catching this, it was a mistake when I was looking for google references to update!

```suggestion
                  - https://github.com/login/oauth/authorize
```